cmd/shell.go

```diff
@@ -187,13 +187,12 @@ func StatsAndLogging(statConf StatsdConfig, logConf SyslogConfig) (statsd.Statte
 	}
 	auditLogger, err := blog.NewAuditLogger(syslogger, stats, level)
 	FailOnError(err, "Could not connect to Syslog")
-	// TODO(https://github.com/cloudflare/cfssl/issues/426):
-	// CFSSL's log facility always prints to stdout. Ideally we should send a
-	// patch that would allow us to have CFSSL use our log facility. In the
-	// meantime, inhibit debug and info-level logs from CFSSL.
-	cfsslLog.Level = cfsslLog.LevelWarning
+
 	blog.SetAuditLogger(auditLogger)
 
+	cfsslLog.Level = cfsslLog.LevelWarning
```

Comment:
My reading of the cfssl code indicates that cfssl will filter messages below this level before even sending them to the syslogger. I think there's no reason to change that to LevelWarning, right? The default of LevelInfo seems fine.